src/Module/Api/Mastodon/InstanceV2.php

```diff
@@ -131,12 +131,19 @@ private function buildConfigurationInfo(): InstanceEntity\Configuration
 
 		return new InstanceEntity\Configuration(
 			$statuses_config,
-			new InstanceEntity\MediaAttachmentsConfig(Images::supportedMimeTypes(), $image_size_limit, $image_matrix_limit),
+			new InstanceEntity\MediaAttachmentsConfig($this->supportedMimeTypes(), $image_size_limit, $image_matrix_limit),
 			new InstanceEntity\Polls(),
 			new InstanceEntity\Accounts(),
 		);
 	}
 
+	private function supportedMimeTypes(): array
+	{
+		$mimetypes = ['audio/aac', 'audio/flac', 'audio/mpeg', 'audio/mp4', 'audio/ogg', 'audio/wav',
```

Comment:
How did you decide on this shortlist?

Reply:
I had a look at the supported media types of the Firefox browser and took the most common ones.

Reply:
But unlike the image formats we support depending on the availability of GD or ImageMagick and their version, we don’t process audio or video files. Why limit the MIME types then?

Reply:
We need to provide specific mimetypes with that endpoint since mobile clients like Tusky fetch it prior to upload media.

Reply:
No wildcards possible like `audio/*` and ˋvideo/*`?

Reply:
As far as I understood the endpoint this isn't possible. Of course we could create some configuration for that.